src/main/java/com/detrav/items/behaviours/BehaviourDetravToolProspector.java

```diff
@@ -256,7 +257,8 @@ protected void processOreProspecting(DetravMetaGeneratedTool01 aItem, ItemStack
                                 addOreToHashMap(name, aPlayer);
                         } else if (DetravScannerMod.isBartWorksLoaded && BartWorksHelper.isOre(tBlock)){
                             if (data != 1 && BartWorksHelper.isSmallOre(tBlock)) continue;
-                            addOreToHashMap(GT_LanguageManager.getTranslation((BartWorksHelper.isSmallOre(tBlock) ? "bw.blockores.02." : "bw.blockores.01.") + ((BartWorksHelper.getMetaFromBlock(aChunk,x,y,z,tBlock))*-1) + ".name"), aPlayer);
+                            final Werkstoff werkstoff = Werkstoff.werkstoffHashMap.getOrDefault((short) ((BartWorksHelper.getMetaFromBlock(aChunk,x,y,z,tBlock))*-1), null);
+                            addOreToHashMap(werkstoff.getLocalizedName() +  " Ore", aPlayer);
```

Comment:
it will make all ores english only.

Reply:
Yes I also had that thought but GT++ also just has the " Ores" suffix added one so I didn't bother

Reply:
try to use `LanguageRegistry.instance().getStringLocalization()`